src/components/info.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import InputWithErrorInfo from './input-error-info';
+
+const INFO  = 'info';
+const NAME  = 'name';
+const PHONE = 'phone';
+const EMAIL = 'email';
+
+export default function Info (props) {
+    const { info = {}, onChange, formErrors, wantEdit, isNewUser } = props;
+    const _onChange                                                = e => onChange(e, INFO);
+    let showNeed                                                   = wantEdit || isNewUser ? '*' : '';
```

Comment:
it's better if `InputWithErrorInfo` decide which mark should be used for required fields.
1. Let's raname prop `showNeed` -> `isEditing` or `showRequiredMark`
1. move `flag ? '*' : '';` into `InputWithErrorInfo`